### PR TITLE
Machine repair changes

### PR DESCRIPTION
**Pull request by tobylane**
**Describe what the proposed change does**
- Move the announcing function to PlayerHospital
- Add the handymen check per the comment

### CODE REVIEW

File: CorsixTH/Lua/hospitals/player_hospital.lua [hunk 1]
```diff
@@ -391,6 +391,19 @@ function PlayerHospital:adviseBoilerBreakdown(broken_heat)
   end
 end
 
+local AnnouncementPriority = _G["AnnouncementPriority"]
```
Comment (lewri) [inline CorsixTH/Lua/hospitals/player_hospital.lua:394]: I would recommend always puttting this line at the top of the file before the class declaration. Also missing the ``require``

File: CorsixTH/Lua/entities/machine.lua [hunk 2]
```diff
@@ -22,8 +22,6 @@ local TH = require("TH")
 
 corsixth.require("announcer")
```
Comment (lewri) [inline CorsixTH/Lua/entities/machine.lua:23]: As per other comment.
You don't need this line anymore if you're taking any announcements out of machine.

**Review by lewri — CHANGES_REQUESTED** [review]
We can ignore the lua failure

**Comment by lewri** [comment]
I still think for the sake of uniformity you should re-add those require and announcementpriority lines into Player Hospital for now.
We can make a separate PR to test removing them across the board if the hunch of them not being required is right.

**Comment by tobylane** [comment]
Agreed. Ready.

**Review by lewri — APPROVED** [review]

**Pull request merged**